web/src/main/java/com/epam/esm/web/advice/CustomErrorCode.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.esm.web.advice;
+
+public enum CustomErrorCode {
+    RECOURSE_EXIST("40901"),
+    RECOURSE_NOT_EXIST("40401"),
+    ILLEGAL_REQUEST_PARAMETER("40001"),
+    ILLEGAL_SORT_PARAMETER("40002"),
+    ERROR_VALIDATION("40003"),
+    ERROR_SQL("40004"),
```

Comment:
ERROR_SQL is not a type of BadRequest code as for me

Reply:
done